src/main/scala/openflights.scala

```diff
@@ -6,15 +6,15 @@ import starcolon.flights.geo.Geo
 import scala.collection.mutable.Map
 
 case class Airline (id: Long, code: String, name: String, country: String){
-  def prettyPrint(){
+  def prettyPrint(): Unit = {
     println("✈️ " + Console.CYAN + name + " (" + code + ") " + Console.WHITE + country + Console.RESET)
   }
 }
 
 case class Airport (code: String, name: String, city: String, country: String, lat: Float, lng: Float){
-  def prettyPrint(){
     println("🏠 " + Console.CYAN + name + " (" + code + ") " + 
       Console.WHITE + city + "/" + country + Console.RESET)
+  def prettyPrint(): Unit = {
```

Comment:
Wait wat. This doesn't compile, the method body moved… I think my diff tool did something weird at the last moment. I'll push a fix in a moment.